manifests/m/Microsoft/WindowsADK/10.1.22000.1/Microsoft.WindowsADK.yaml

```
@@ -0,0 +1,26 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+PackageIdentifier: Microsoft.WindowsADK
+PackageVersion: 10.1.22000.1
+PackageName: Windows Assessment and Deployment Kit 
+Publisher: Microsoft Corporation
+License: Copyright (c) Microsoft Corporation
+Moniker: adk
+Tags:
+- adk
+- sccm
+- mdt
+- windows
+ShortDescription: The Windows Assessment and Deployment Kit (Windows ADK) has the tools you need to customize Windows images for large-scale deployment, and to test the quality and performance of your system, its added components, and the applications running on it.
+PackageUrl: https://docs.microsoft.com/en-us/windows-hardware/get-started/adk-install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://download.microsoft.com/download/1/f/d/1fd2291e-c0e9-4ae0-beae-fbbe0fe41a5a/adk/adksetup.exe
+  InstallerSha256: D3922B8F37AAC5F74CA367BEDD3B0318EF045B1BB58EAD3134776E6C7AB1C882
+  InstallerType: exe
```

mmmmmm i really want this change, but ill catch it another time
```suggestion
  InstallerType: burn
```